.github/workflows/ci.yml

```diff
@@ -532,23 +426,19 @@ jobs:
   test_dockerfile:
     runs-on: ubuntu-latest-m
     needs: [ setup, lint ]
+    strategy:
+      matrix:
+        python-version: [ "3.8","3.9","3.10","3.11" ]
```

Comment:
why are you adding this?

Reply:
note this is the dockerfile test...

Reply:
So we not want it being tested against all python versions we support?

Reply:
it's in a docker container, I don't think the python version would affect it? unless I'm missing something

Reply:
Forgot we’re using our own base image, I had thought we were dynamically building it off a base python image in which case this would make sense. Will revert.